docker_setup.sh

```diff
@@ -145,4 +145,5 @@ docker run --name bess-web -d --restart unless-stopped \
 docker run --name bess-cpiface -td --restart unless-stopped \
 	--net container:bess \
 	--entrypoint zmq-cpiface \
-	upf-epc-cpiface:"$(<VERSION)"
+	upf-epc-cpiface:"$(<VERSION)" -u 198.18.0.1 \
+	-N 172.17.0.1 -Z 172.17.0.2
```

Comment:
use the long option

Reply:
install.md update
Should we have a combined json config for cpiface and bess? let both initialize off of it?